cross/build-rootfs.sh

```diff
@@ -16,7 +15,6 @@ __InitialDir=$PWD
 __BuildArch=arm
 __UbuntuArch=armhf
 __UbuntuRepo="http://ports.ubuntu.com/"
-__LLDB_Package="lldb-3.6-dev"
```

Comment:
There's no liblldb-dev for arm64 for Ubuntu 16.10 and earlier (it is present in 17.04, though). I'll revert this change and add the no-lldb option. This makes it consistent with coreclr and corefx.